templates/partials/header.php

```diff
@@ -6,9 +6,40 @@
 
 /** @var string $html_title */
 /** @var string|callable $page_title */
+/** @var string $url */
+/** @var string $image_url */
+/** @var string $html_description */
 /** @var ?callable $sub_head */
 /** @var ?string[] $breadcrumbs */
 
+$html_title       = $html_title ?? esc_html__( 'Translation Events', 'gp-translation-events' );
+$url              = $url ?? URLS::events_home();
+$html_description = $html_description ?? esc_html__( 'WordPress Translation Events', 'gp-translation-events' );
+$image_url        = $image_url ?? URLs::event_default_image();
+
+add_action(
+	'gp_head',
+	function () use ( $html_title, $url, $html_description, $image_url ) {
+		echo '<meta name="twitter:card" content="summary" />' . "\n";
```

Comment:
Instead of echos, we could do:

```php
function () use ( $html_title, $url, $html_description, $image_url ) {
?>
    <meta name="twitter:title" content="<?php echo esc_attr( $html_title ); ?>" />
<?php
```

Reply:
For me, it is more readable as it is, so I prefer to leave it as it is, but it is entirely up to opinion. Thanks for the feedback.

Reply:
No problem, just a suggestion.